docs/blog/crucial-tests/index.md

```diff
@@ -394,6 +392,50 @@ beforeAll(() => {
 });
 ```
 
+## 📦 Test the package as a consumer
+
+a lot of times we don't test our packages as how our consumers will use them,
+this is a simplified example of how we could do that
+
+```js
+// global-setup.js
```

Comment:
I wouldn't coupld to Jest, before hook is more universal

Reply:
global setup is not coupled to jest, is to show that it happen only once - hence global